mobile-app/cypress/integration/functional/wallet/loans/loans.spec.ts

```diff
@@ -350,6 +350,11 @@ context('Wallet - Loans - Payback Loans', () => {
     cy.getByTestID('conversion_info_text').should('not.exist')
   })
 
+  it('should not display warning msg when input is NaN or less than 0', function () {
+    cy.getByTestID('payback_input_text').clear()
```

Comment:
you handled the `NaN` case, but your test title also mention less than 0, want to add a step for that?